<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateTicket</class>
 <widget class="QMainWindow" name="UpdateTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="edit_field_destiny">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_field_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_seat">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seat</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_box_id_flight">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_origin">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Origin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_origin">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit the desired fields</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_date">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="time_flight">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_destiny">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Destiny</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_box_number_seat">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>900</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_id_flight">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Flight ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_time">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_ticket">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>498</x>
      <y>470</y>
      <width>261</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_update_ticket">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
